kalite/caching/__init__.py

```diff
@@ -1,99 +0,0 @@
-"""
```

Comment:
I believe so. Searching for CACHE_VARS should show any instances.

On Tue, 15 Sep 2015 09:04 Aron Fyodor Asor <notifications@github.com> wrote:

> In kalite/caching/__init__.py
> <https://github.com/learningequality/ka-lite/pull/4261#discussion_r39529372>
> :
>
> > @@ -1,99 +0,0 @@
> > -"""
>
> Do we only use the in-memory caches for i18n and topic tool-related
> functions?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/learningequality/ka-lite/pull/4261/files#r39529372>.
>